src/app/views/layout.controller.coffee

```diff
@@ -17,10 +17,13 @@ angular.module 'mnoEnterpriseAngular'
     $scope.$watch(MnoeOrganizations.getSelectedId, (newValue) ->
       MnoeCurrentUser.get().then(
         (response) ->
+          selectedOrg = _.find(response.organizations, {id: newValue})
+          isAdmin = MnoeOrganizations.role.atLeastAdmin(selectedOrg.current_user_role)
+          $scope.isCartEnabled = if isAdmin then MnoeConfig.isProvisioningEnabled() else false
```

Comment:
Could be refactored to:
`$scope.isCartEnabled = isAdmin && MnoeConfig.isProvisioningEnabled()`

Reply:
Good combining the logic from https://github.com/maestrano/mno-enterprise-angular/pull/507
and good catch on the appManagement part of it. Don't want that flag to control whether or not the cart is enabled.